src/api.rs

```diff
@@ -741,6 +786,10 @@ impl<T: Pixel> Context<T> {
     (fi, true)
   }
 
+  /// Receive packet functions provide an Encode API with decoupled packets/frames, On sucess it returns packet with compressed frame.
```

Comment:
Do not mention implementation details that may and should be changed soon :)

"Returns an encoded packet or signals that additional frame data must be provided"

Mention `flush()` and `send_frame()` .